### PR TITLE
Create 2018-12-18-prestashop-1-7-5-0-available.md

### PR DESCRIPTION
**Pull request by ttoine**

### CODE REVIEW

File: news/_posts/releases/2018-12-18-prestashop-1-7-5-0-available.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
+---
+layout: post
+title:  "PrestaShop 1.7.5.0 is available!"
+subtitle: "Improvements and bugs fixes under the hood, and exciting new features"
```
Comment (matks) [inline news/_posts/releases/2018-12-18-prestashop-1-7-5-0-available.md:4]: - `bugs fixes`=> `bug fixes` ? (not sure)
- twice the usage of "and" ?

File: news/_posts/releases/2018-12-18-prestashop-1-7-5-0-available.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+  - Bosnian
+  - Mexican Spanish
+  - Latvian
+  - Galician
+  - Arabic
+
+
+## The technical side
+
+### Bugs fixes
```
Comment (matks) [inline news/_posts/releases/2018-12-18-prestashop-1-7-5-0-available.md:88]: `Bugs fixes` or `Bug fixes`(not sure) ?

File: news/_posts/releases/2018-12-18-prestashop-1-7-5-0-available.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+- action{GridId}GridFilterFormModifier
+- action{GridId}GridPresenterModifier
+- actionFrontControllerSetVariables
+
+The developer documentation includes a [description of hooks](https://devdocs.prestashop.com/1.7/modules/concepts/hooks/list-of-hooks/).
+
+
+### Clear cache on module action
+
+PrestaShop now clears the symfony cache when an action is performed on a module (install, uninstall, enable, …) This is necessary because modules can now use Symfony routing and services, so we need to clear the cache if we want these services/routing to be updated.
```
Comment (matks) [inline news/_posts/releases/2018-12-18-prestashop-1-7-5-0-available.md]: Missing dot after `(install, uninstall, enable, …)`

File: news/_posts/releases/2018-12-18-prestashop-1-7-5-0-available.md [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+
+
+### Changes
+
+A lot has been done to make this “minor” PrestaShop version a reality. Here’s 1.7.5 in numbers:
+
+- 448 merged Pull Requests (vs 210 in 1.7.3 and 260 in 1.7.4)
+- files changed (164812 insertions, 129497 deletions)
```
Comment (matks) [inline news/_posts/releases/2018-12-18-prestashop-1-7-5-0-available.md]: Missing number

File: news/_posts/releases/2018-12-18-prestashop-1-7-5-0-available.md (hunk 3 above)
Comment (ttoine) [reply news/_posts/releases/2018-12-18-prestashop-1-7-5-0-available.md]: done

File: news/_posts/releases/2018-12-18-prestashop-1-7-5-0-available.md (hunk 4 above)
Comment (ttoine) [reply news/_posts/releases/2018-12-18-prestashop-1-7-5-0-available.md]: done

**Pull request merged**